packages/react/src/components/heliopayContainer/constants.ts

```diff
@@ -0,0 +1,21 @@
+import { Currency, CustomerDetails, ProductDetails } from '@heliofi/common';
+
+export type SubmitPaymentProps = {
```

Comment:
why is the type in cosntants file ?

Reply:
this isn't a component prop, but for a method. do you see any issues?

Reply:
I would still not put it in costsnts it's not a const

Reply:
Types file?